src/lib/bits/select/components/SelectTrigger.svelte

```diff
@@ -22,6 +23,7 @@
 		use:melt={builder}
 		{...$$restProps}
 		{...attrs}
+		{type}
```

Comment:
```suggestion
		type="button"
```

Reply:
@huntabyte all done. And also more... I searched for button elements within bits components and added more `type=button`. I'm pretty sure all of them needs it.

Especially checkbox or radio - you don't want form inputs to trigger submit at any time. Same with tabs, popovers, etc.

Reply:
You're incredible, thank you!